src/tests/intg/test_secrets.py

```diff
@@ -100,11 +100,11 @@ def setup_for_secrets(request):
         [secrets]
         max_secrets = 10
         max_payload_size = 2
+        client_idle_timeout = 10
```

Comment:
Is there a reason why there need to be 10 seconds?
Because I would like to avoid longed sleeps in tests `time.sleep(15)`

And IMHO we should use separate sssd.conf for this test. Because default value should be used for standard secrets tests.